django/db/models/query.py

```diff
@@ -1538,7 +1538,12 @@ def prefetch_one_level(instances, prefetcher, lookup, level):
     # contains some prefetch_related lookups. We don't want to trigger the
     # prefetch_related functionality by evaluating the query. Rather, we need
     # to merge in the prefetch_related lookups.
-    additional_lookups = getattr(rel_qs, '_prefetch_related_lookups', [])
+    # We need to copy the lookups in case it is a Prefetch object which could
+    # be reused later. See #24873.
```

Comment:
If possible, try to capture the essence of the ticket in the comment so we don't need the reference the ticket. In general we try to avoid ticket references in the code for all but the most obscure issues. Ticket references in tests are more acceptable though.